tasks/src/tabular-classification/about.md

```diff
@@ -15,18 +15,41 @@ Tabular classification models can be used in detecting fraudulent credit card tr
 ### Churn Prediction
 Tabular classification models can be used in predicting customer churn in telecommunication. An example dataset for the task is hosted [here](https://huggingface.co/datasets/scikit-learn/churn-prediction).
 
-## Model Hosting and Inference
 
-You can use the [skops](https://skops.readthedocs.io/en/stable/) library to share, explore, and use `scikit-learn` models on the Hugging Face Hub. `skops` models have widgets to try the models on the browser and have descriptive reports (also known as model cards) in their repositories. You can pull a `scikit-learn` model like below using `skops`:
+# Model Hosting and Inference 
 
+You can use [skops](https://skops.readthedocs.io/) for model hosting and inference on 🤗Hub. Using `skops` you can:
+- Easily use inference API,
+- Build Gradio UIs with one line of code,
+- Programmatically create model cards,
+- Securely serialize your scikit-learn model. (See limitations of using pickle [here](https://huggingface.co/docs/hub/security-pickle).)
```

Comment:
Not sure all the audience will know what scikit-learn is :thinking:

Reply:
I will put a note but it really is in 101 of these tasks.

Reply:
addressed this.